llarp/handlers/tun.cpp

```diff
@@ -35,8 +35,13 @@ namespace llarp
     void
     TunEndpoint::tunifTick(llarp_tun_io *tun)
     {
-      auto *self = static_cast< TunEndpoint * >(tun->user);
-      LogicCall(self->m_router->logic(), [self]() { self->Flush(); });
+      auto *self     = static_cast< TunEndpoint * >(tun->user);
+      const auto now = self->Now();
```

Comment:
so the libuv event loop snapshots a "now" every event loop tick that could be used here (tun->loop->whatever or something like that).  idk if the current Windows event loop supports that concept though.